dev/Detours/README.md

```diff
@@ -0,0 +1,6 @@
+This is a fork of Detours from https://github.com/microsoft/Detours
+
+Last forked
+-----------
+commit  e9da64c72f293c0223f293f018067fe68fbdb801
```

Comment:
Per thread with @jevansaks - is there any hope of taking a source package here, rather than dumping the entire thing in place?  Even if we were to produce the package and host it somewhere as an artifact for reference by our build, for instance.

Reply:
We can offer a patch to the Detours project to make a nuget package. That'll require a .nuspec, an update to Detours' build pipeline (or creating one if they don't have?) and offering a feed

Worth doing...later. Rolling with this for now to unblock work. We can replace later with a Detours nuget when they create one (or we offer a patch doing it)

Are we tracking these sorts of TODO-vFuture tasks via github or ADO?

Reply:
@EHO-makai do you know which is preferred github vs ADO Q for TODO-vFuture tasks?

Reply:
For code that is living on GitHub I'd say we should track it on GitHub. That way there's even a chance an awesome community member might help us out!

Reply:
Done => https://github.com/microsoft/ProjectReunion/issues/178